src/helpers/approvals-satisfied.ts

```diff
@@ -65,4 +65,14 @@ export const approvalsSatisfied = async ({ teams, number_of_reviewers = '1', pul
   return booleans.every(Boolean);
 };
 
-const createArtificialCodeOwnersEntry = (teams: string[]) => teams.map(team => ({ owners: [team] }));
+const createArtificialCodeOwnersEntry = (teams?: string[], users?: string[]) => [{ owners: (teams || []).concat(users || []) }];
```

Comment:
Could we destructure the arguments like this?
```suggestion
const createArtificialCodeOwnersEntry = ({ teams = [], users = [] }: { teams?: string[], users?: string[] }) => [{ owners: teams.concat(users) }];
```

Reply:
I have very less experience in typescript :P
If you say the above works, then it's fine 💯  UTs will verify anyway if it's working on it